api/serializers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import serializers
+from .models import address, cityID, stateID
+
+class AddressGetSerializer(serializers.ModelSerializer):
+    #Added 2 additional fields which are derived from related 'cityID' and 'stateID' models
+    city_name = serializers.CharField(source='city.city_name', read_only=True)
+    state_name = serializers.CharField(source='state.state_name', read_only=True)
+    
+    class Meta:
+        model = address
+        fields = ('id', 'username', 'line1', 'line2', 'city_name', 
```

Comment:
maybe move each entry on it's own line? easier to read and future addition/deletion would produce better diffs.